app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt

```diff
@@ -175,6 +176,9 @@ class ProjectPageActivity :
             env
         }
 
+        val ffClient = requireNotNull(environment?.featureFlagClient())
+        ffClient.activate(this)
```

Comment:
calling `activate` when the activity is loaded will minimize the fist installation gap where the information requested to firebase remote config has not yet been applied.